src/seedu/addressbook/commands/assessment/AddAssignmentStatistics.java

```diff
@@ -44,10 +59,45 @@ public AssignmentStatistics getAssignmentStatistics() {
     @Override
     public CommandResult execute() {
         try {
+            final Map<Person, Grades> grade;
+            Assessment assessName = getTargetAssessment();
+            examName = assessName.getExamName();
+            double maxGrade = 0;
+            double minGrade = 1000000;
+            double total = 0;
+            int count = 0;
+            grade = assessName.getAllGrades();
+            for (Grades gradeVal : grade.values()) {
+                if (gradeVal.getValue() > maxGrade) {
+                    maxGrade = gradeVal.getValue();
+                }
+                if (gradeVal.getValue() < minGrade) {
+                    minGrade = gradeVal.getValue();
+                }
```

Comment:
Pretty sure Java have the max/min functions like in C++, could consider using them